src/push-notifications.js

```diff
@@ -2,11 +2,17 @@ import doRequest from './doRequest';
 import TokenProvider from './token-provider';
 import DeviceStateStore from './DeviceStateStore';
 
+const DEFAULT_SERVICE_WORKER_URL = '/sw.js';
```

Comment:
Perhaps a bit more explicit default name?

```suggestion
const DEFAULT_SERVICE_WORKER_URL = '/service-worker.js';
```